Library/Homebrew/cmd/uninstall.rb

```diff
@@ -29,15 +32,25 @@ def uninstall_args
   def uninstall
     uninstall_args.parse
 
-    kegs_by_rack = if args.force?
-      Hash[args.named.map do |name|
+    if args.force?
+      casks = []
+      kegs_by_rack = Hash[args.named.map do |name|
```

Comment:
I'd consider just populating the Hash with a loop; the `Hash[...do` reads a bit weird.